templates/cloud/cloud-computing.html

```diff
@@ -0,0 +1,494 @@
+    </div>
+  </div>
+</section>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-5">
+      <h2>Cloud computing vs virtualisation</h2>
+      <h4>Even though cloud computing uses virtualisation underneath, it is much more than that. While virtualisation is more like an engine, cloud computing is more like a car.</h4>
+    </div>
+    <div class="col-7">
+      <p>People who are new to cloud computing often get confused with the cloud computing vs virtualisation dilemma. Virtualisation is a technology that divides the physical resources of a server, such as the central processing unit (CPU), random access memory (RAM), storage and network, into chunks. Those can further be aggregated into VMs, consisting of virtual resources and allocated to end users. Hardware virtualisation, VM creation and termination is handled by a dedicated software, firmware or hardware component called a <a href="/blog/kvm-hyphervisor">hypervisor</a>.</p>
+      <p>In turn, cloud computing expands the capabilities provided by the hypervisor. The cloud software manages all the hypervisors and automates the VM provisioning process. Whenever a cloud user requests a VM with specific parameters through the self-service portal, it tries to find the least utilised hypervisor (unless configured otherwise), schedules VM creation on this hypervisor and returns VM details back to the user. Likewise, it automates the creation of other types of virtual resources, such as block storage devices, virtual networks, and more.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Cloud computing benefits</h2>
+      <p class="u-sv3">Compared to legacy IT infrastructure based on mainframes, blades and "pet” servers, cloud computing transforms data centres into resource pools, providing end users with immediate access to hardware: <a href="https://maas.io">physical</a> or <a href="/openstack/features">virtual</a>. The primary benefits of cloud computing include:</p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4">
+      <div class="u-sv1 u-hide--small">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/4d3a4582-edge-cluster.svg",
+        alt="Long-term cost savings",
+        width="64",
+        height="64",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
```

Comment:
This image is smaller than the others

Reply:
Yep sorry, it was supposed to be the next one.